src/docs/README.md

```diff
@@ -137,31 +156,53 @@ Any file named `README.md` (case sensitive!) will not be used to generate html p
 
 ## Structure and Format
 
-- Max. 3 headline levels (## & ###)
+Try to avoid deep structures by using no more than two headline levels, i.e.
```

Comment:
Not sure if we need to enforce this, but that is fine with me if you want this 👍